.github/workflows/continuous-integration.yml

```diff
@@ -17,14 +28,11 @@ jobs:
 
       - uses: nanasess/setup-php@master
         with:
-          php-version: 8.1
+          php-version: 8.2
```

Comment:
Is dit nodig? Ik vind het wel fijn als de php versie blijft op de oudste versie die we ondersteunen.